app/main.py

```diff
@@ -1,44 +1,70 @@
 import time
+import asyncio
 
 from iot.devices import HueLightDevice, SmartSpeakerDevice, SmartToiletDevice
 from iot.message import Message, MessageType
 from iot.service import IOTService
 
 
-def main() -> None:
+async def main() -> None:
     # create an IOT service
     service = IOTService()
 
     # create and register a few devices
     hue_light = HueLightDevice()
     speaker = SmartSpeakerDevice()
     toilet = SmartToiletDevice()
+
     hue_light_id = service.register_device(hue_light)
     speaker_id = service.register_device(speaker)
     toilet_id = service.register_device(toilet)
 
-    # create a few programs
-    wake_up_program = [
-        Message(hue_light_id, MessageType.SWITCH_ON),
-        Message(speaker_id, MessageType.SWITCH_ON),
-        Message(speaker_id, MessageType.PLAY_SONG, "Rick Astley - Never Gonna Give You Up"),
-    ]
+    hue_light_id, speaker_id, toilet_id = await asyncio.gather(
+        hue_light_id, speaker_id, toilet_id
+    )
 
```

Comment:
What are doing this code?

Reply:
https://github.com/mate-academy/py-asyncio-iot/pull/240#discussion_r1434129229
I use await asyncio.gather here and get ids to use in my code